tests/integration/test_apidoc.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from fastapi.openapi.utils import get_flat_models_from_routes
+from fastapi.utils import get_model_definitions
+from pydantic.schema import get_model_name_map
+from starlette.testclient import TestClient
+
+logger = logging.getLogger(__name__)
+
+
+def test_apidoc(client: TestClient) -> None:
+    # Asserts that the apidoc can be loaded
+    flat_models = get_flat_models_from_routes(client.app.routes)
+    model_name_map = get_model_name_map(flat_models)
+    try:
+        get_model_definitions(flat_models=flat_models, model_name_map=model_name_map)
+    except Exception as e:
```

Comment:
Actually, the raised exception is `ValidationError`.
It's better not to catch the exception at all, and let the test raises.

Reply:
Here is what we get when the validation error is raised:
```python
tests/integration/test_apidoc.py:11: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
venv/lib/python3.8/site-packages/fastapi/utils.py:24: in get_model_definitions
    m_schema, m_definitions, m_nested_models = model_process_schema(
pydantic/schema.py:602: in pydantic.schema.model_process_schema
    ???
antarest/study/business/areas/st_storage_management.py:48: in schema_extra
    schema["example"] = STStorageInput(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   pydantic.error_wrappers.ValidationError: 1 validation error for STStorageInput
E   foo
E     extra fields not permitted (type=value_error.extra)

pydantic/main.py:341: ValidationError
```

Reply:
Indeed, it's way better